app/views/admin/courses/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with model: [:admin, @course],
+              local: true do |f| %>
+  <div class="inline-labels">
+    <%= f.label :name,
+                Course.human_attribute_name(:name),
+                class: 'required' %>
+    <%= f.text_field :name,
+                     size: 35,
+                     required: true %>
+    <%= f.label :display_name,
+                Course.human_attribute_name(:display_name),
+                class: 'required' %>
+    <%= f.text_field :display_name,
+                     size: 35,
+                     required: true %>
+    <%= f.label :is_hidden, Course.human_attribute_name(:is_hidden) %>
+    <span>
+      <%= f.label :is_hidden, I18n.t('assignments.section_hidden.hidden'), value: 'true' %>
+      <%= f.radio_button :is_hidden, true %>
+      <%= f.label :is_hidden, I18n.t('assignments.section_hidden.visible'),  value: 'false' %>
+      <%= f.radio_button :is_hidden, false %>
+    </span>
+  </div>
+  <p>
+    <%= f.submit t(:save),
+                 data: { disable_with: t('working') } %>
+    <%= link_to t(:cancel), admin_courses_path, class: 'button' %>
```

Comment:
Not a bad idea, but our other forms don't have such a button, so to be consistent you should probably remove it.